job-service-db/src/main/resources/procedures/dropTables.sql

```diff
@@ -0,0 +1,45 @@
+--
+-- Copyright 2016-2021 Micro Focus or one of its affiliates.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ *  Name: drop_deleted_task_tables
+ *
+ *  Description:
+ *  Drops all tables present in delete_log table
+ *  This procedure does batch commits. The batch is defined by commit_limit variable. Default batch size being 10.
+ */
+CREATE OR REPLACE PROCEDURE drop_deleted_task_tables()
+LANGUAGE plpgsql
```

Comment:
Should volatility categories be specified for procedures?  Or does the concept only exist for functions?

Reply:
Don't do it in this PR but now that we have both procedures and functions I'd be tempted to have a follow on separate them out into different folders.  I can easily see the distinction being lost with them all just sitting in the same folder.

Reply:
I did not find any mention of volatility categories w.r.t procedures in PG documentation. Got below error if I create a procedure with volatile keyword.
![image](https://user-images.githubusercontent.com/4994904/112939267-cc86ef00-9148-11eb-9511-ea05b4c6f960.png)
The screenshot does not show the cursor but it pointed at the volatile keyword at the time of error.